fml/message_loop_task_queues_benchmark.cc

```diff
@@ -14,7 +13,7 @@
 namespace fml {
 namespace benchmarking {
 
-static void BM_RegisterAndGetTasks(benchmark::State& state) {
+static void BM_RegisterAndGetTasks(benchmark::State& state) {  // NOLINT
```

Comment:
Since this is a static function, fixing this should be easy.

Reply:
Oh, yeah, didn't notice that... :-)

Reply:
Actually, not so simple, since it uses the BENCHMARK macro, so that entire API would have to change.